services/catalog/src/simcore_service_catalog/api/rpc/_services.py

```diff
@@ -75,11 +76,14 @@ async def list_services_paginated(
     assert len(items) <= total_count  # nosec
     assert len(items) <= limit  # nosec
 
-    return PageRpcServicesGetV2.create(
-        items,
-        total=total_count,
-        limit=limit,
-        offset=offset,
+    return cast(
+        PageRpcServicesGetV2,
+        PageRpcServicesGetV2.create(
+            items,
+            total=total_count,
+            limit=limit,
+            offset=offset,
+        ),
```

Comment:
you can also do it via typing
```suggestion
    result: PageRpcServicesGetV2 = PageRpcServicesGetV2.create(
            items,
            total=total_count,
            limit=limit,
            offset=offset,
        )
    )
    return result
```